nexus/src/app/background/tasks/region_snapshot_replacement_garbage_collect.rs

```diff
@@ -97,13 +97,7 @@ impl RegionSnapshotReplacementGarbageCollect {
                         ok for {request_id}"
                     );
 
-                    info!(
-                        &log,
-                        "{s}";
-                        "request.snapshot_id" => %request.old_snapshot_id,
-                        "request.region_id" => %request.old_region_id,
-                        "request.dataset_id" => %request.old_dataset_id,
-                    );
+                    info!(&log, "{s}");
```

Comment:
I like more info (like snapshot/region/dataset), can we print the whole struct?

Reply:
definitely, done in 3ec534122 :)